src/server.py

```diff
@@ -0,0 +1,38 @@
+"""Implementation of server."""
+
+import socket
+
+
+def server():
+    """Create a server."""
+    address = ('127.0.0.1', 5000)
+    server = socket.socket(socket.AF_INET,
+                           socket.SOCK_STREAM,
+                           socket.IPPROTO_TCP)
+    server.bind(address)
+    server.listen(1)
+    try:
+        while True:
+            conn, addr = server.accept()
+            message = message_handle(conn)
+            conn.sendall(message.encode('utf8'))
+
+    except (KeyboardInterrupt, BrokenPipeError):
+        print ('Shutting Down')
+        server.close()
+
+
+def message_handle(message):
+    """Handle input message relative to buffer length."""
+    buffer_length = 8
+    message_complete = False
+    output = ''
```

Comment:
This could be problematic since this string literal will be unicode in Python 3 and bytestring in Python 2. Better to be explicit and force it to be consistent in both versions.